requirements.txt

```diff
@@ -1,55 +1,65 @@
-amqp==1.4.9
-anyjson==0.3.3
-appdirs==1.4.3
-arrow==0.4.2
-Babel==2.3.4
-billiard==3.3.0.23
-bleach==2.1.3
-boto==2.38.0
-boto3==1.7.3
-botocore==1.10.3
-CairoSVG==2.1.3
-celery==3.1.25
-certifi==2017.7.27.1
-chardet==3.0.4
-dj-database-url==0.4.2
-Django==1.11.6
-django-appconf==1.0.1
-django-crispy-forms==1.6.1
-django-phonenumber-field==1.1.0
-django-webpack-loader==0.5.0
-docutils==0.14
-filechunkio==1.6
-freezegun==0.3.8
-futures==2.2.0
-geonamescache==0.21
-gunicorn==19.7.1
-html5lib==1.0.1
-httplib2==0.9.1
-icalendar==4.0.3
-idna==2.6
-jmespath==0.9.3
-kombu==3.0.37
-mock==1.0.1
-olefile==0.44
-packaging==16.8
-phonenumberslite==7.7.0
-Pillow==4.3.0
-psycopg2==2.7.3.1
-pygal===2.4.0
-PyJWT==1.5.3
-pyparsing==2.2.0
-PySocks==1.6.8
-python-dateutil==2.6.1
-pytz==2017.2
-requests==2.18.4
-s3-backup-rotate==0.3.3
-s3transfer==0.1.13
-six==1.11.0
-transifex-client==0.12.4
-twilio==6.8.0
-unicodecsv==0.14.1
-urllib3==1.22
-vine==1.1.3
-webencodings==0.5.1
-
+amqp==1.4.9
+anyjson==0.3.3
+appdirs==1.4.3
+arrow==0.4.2
+Babel==2.3.4
+beautifulsoup4==4.6.3
+billiard==3.3.0.23
+bleach==2.1.3
+boto==2.38.0
+boto3==1.7.3
+botocore==1.10.3
+cairocffi==0.9.0
+CairoSVG==2.1.3
+celery==3.1.25
+certifi==2017.7.27.1
+cffi==1.11.5
+chardet==3.0.4
+cssselect2==0.2.1
+cssutils==1.0.2
+defusedxml==0.5.0
+dj-database-url==0.4.2
+Django==1.11.6
+django-appconf==1.0.1
+django-crispy-forms==1.6.1
+django-inlinecss==0.1.2
+django-phonenumber-field==1.1.0
+django-premailer==0.2.0

```

Comment:
I'd much rather we don't add this dependency. The last update was 2 years ago and it isn't used by many people. https://github.com/alexhayes/django-premailer. Rather use premailing directly: https://github.com/peterbe/premailer or cut it completely

Reply:
ok i'll update it